### PR TITLE
README.md: add slack link for collaboration

### PR DESCRIPTION
**Pull request by kenplusplus**

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 
 # Welcome to Confidential Computing API Group
 
+Collaboration at [slack](https://app.slack.com/huddle/T070GF9PKTQ/C0708HZ9087)
```
Comment (ruomengh) [inline profile/README.md]: I'm not sure if this is the correct link. It seems a cc-api huddle but not the channel.

Comment (kenplusplus) [reply profile/README.md]: fixed at https://github.com/cc-api/.github/pull/1/commits/d09b6e071a9ef80883684d4f7a0b07492f1b14f5

**Review by ruomengh — APPROVED** [review]
LGTM

File: profile/README.md [hunk 2]
```diff
@@ -43,3 +45,8 @@ _NOTE: Please refer to https://www.redhat.com/en/blog/confidential-computing-use
 - [Full Disk Encryption](https://github.com/cc-api/full-disk-encryption)
 - [Confidential Cloud Native AI Pipeline](https://github.com/intel/cloud-native-ai-pipeline/blob/main/docs/How_to_Protect_AI_Models_in_Cloud_Native_Environments.md)
 - [Confidential Cluster](https://github.com/cc-api/confidential-cluster)
+
+# 4. Community
+
+- 2024/05/07 [CCC Attestation SIG] Q&A Update f ([Video](https://www.youtube.com/watch?v=qoAOXZpdz38), Slide)
```
Comment (ruomengh) [inline profile/README.md]: Is the extra "f" a typo?

Comment (kenplusplus) [reply profile/README.md]: sorry for typo, fixed at https://github.com/cc-api/.github/pull/1/commits/9818a2b53f0e1cb9881f173b4d99235af5fcfec6

**Pull request merged**